src/core/broker_config.h

```diff
@@ -17,14 +17,15 @@
  * https://github.com/ETCLabs/RDMnetBroker
  *****************************************************************************/
 
-#ifndef _BROKER_CONFIG_H_
-#define _BROKER_CONFIG_H_
+#ifndef BROKER_CONFIG_H_
```

Comment:
Super minor question -- was there a reason for this change?

Reply:
Strictly speaking, names that start with one underscore and a capital letter, or contain two underscores, are reserved by the C and C++ standards for compiler and standard library implementations.
https://en.wikipedia.org/wiki/Include_guard#Discussion
http://forums.codeguru.com/showthread.php?459780-Naming-conventions-(include-guards)

Reply:
Makes sense, I have that ingrained for historical reasons.  I'll have to remember this..